homeassistant/components/remote/harmony.py

```diff
@@ -347,6 +349,9 @@ def new_activity(self, activity_info: tuple) -> None:
 
         num_repeats = kwargs.get(ATTR_NUM_REPEATS)
         delay_secs = kwargs.get(ATTR_DELAY_SECS, self._delay_secs)
+        hold_secs = kwargs.get(ATTR_HOLD_SECS)
+        _LOGGER.debug("Sending commands to device %s holding for %s seconds",
+                      device, hold_secs)
```

Comment:
Why not log `delay_secs` as well?

Reply:
Cause I do not use it? :-)

Makes perfect sense, will add it.